plugins/quickinspector/quickinspector.cpp

```diff
@@ -243,9 +243,9 @@ static QString qsgTextureWrapModeToString(QSGTexture::WrapMode wrapMode)
 
 static bool isGoodCandidateItem(QQuickItem *item)
 {
-    if (!item->isVisible() || item->z() < 0 || qFuzzyCompare(item->opacity() + 1.0, qreal(1.0)) ||
```

Comment:
Can't we make those properties instead of removing them ?
Those were asked features by kdabians on different projects.